common/src/main/java/com/revenuecat/purchases/common/FileHelper.kt

```diff
@@ -0,0 +1,63 @@
+package com.revenuecat.purchases.common
+
+import android.content.Context
+import java.io.BufferedReader
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileOutputStream
+import java.io.InputStreamReader
+
+class FileHelper(
+    private val applicationContext: Context,
+) {
+    fun appendToFile(filePath: String, contentToAppend: String) {
+        val file = getFileInFilesDir(filePath)
+        file.parentFile?.mkdirs()
+        val outputStream = FileOutputStream(file, true)
+        outputStream.use {
+            outputStream.write(contentToAppend.toByteArray())
+        }
+    }
+
+    fun deleteFile(filePath: String): Boolean {
+        val file = getFileInFilesDir(filePath)
+        return file.delete()
+    }
+
+    fun readFilePerLines(filePath: String): List<String> {
+        val readLines = mutableListOf<String>()
+        val file = getFileInFilesDir(filePath)
+        FileInputStream(file).use { fileInputStream ->
+            InputStreamReader(fileInputStream).use { inputStreamReader ->
+                BufferedReader(inputStreamReader).use { bufferedReader ->
+                    readLines.addAll(bufferedReader.readLines())
+                }
+            }
+        }
+        return readLines
+    }
+
+    fun removeFirstLinesFromFile(filePath: String, numberOfLinesToRemove: Int) {
+        val readLines = readFilePerLines(filePath)
+        deleteFile(filePath)
+        val textToAppend = if (readLines.isEmpty() || numberOfLinesToRemove >= readLines.size) {
+            ""
```

Comment:
I'm curious about this case. 
This should in theory never happen because we're calling this only when we know we have a few lines in place already that we want to delete, right?

Reply:
Well, for our current use case that's correct, but since this is a helper which might end up being used by other classes in the future, this should handle those cases. I guess we could throw in those cases, but this seems safer.

Reply:
if we do decide to not crash, we should probably at least log something and / or state in a comment what has happened here, since it's not expected for us to get here

Reply:
Added log